components/brave_private_cdn/private_cdn_helper_unittest.cc

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_private_cdn/private_cdn_helper.h"
+
+#include "testing/gtest/include/gtest/gtest.h"
+
+TEST(BravePrivateCdnHelper, RemovePadding) {
```

Comment:
these tests look OK to me - but would there be value in implementing the same tests as in https://github.com/brave/simplepadding/blob/master/tests/decode.py ?

Reply:
They are actually the same tests :) Did you see one I missed or something like that?

The ones in C++ are harder to read because of the way that `clang-format` puts multiple parts of the same string on different lines.

Reply:
oh wow- yeah I had missed that because of the formatting 😄 Had to scroll ALL the way over to see all the aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa's 😂